flight_planning/v1/telemetry.yaml

```diff
@@ -0,0 +1,220 @@
+      required:
+        - timestamp
+      type: object
+      properties:
+        id:
+          description: >-
+            Unique identifier for this aircraft state/telemetry.  If a USS has already queued an aircraft state with this ID as telemetry to be delivered, this state may be ignored.
+          type: string
+        timestamp:
+          description: >-
+            Time at which this state is valid.  This is equivalent to the time of sensor measurement, so the USS's primary system under test should not be aware of this state until after this time.
+          anyOf:
+            - $ref: 'geotemporal.yaml#/components/schemas/Time'
+        timestamp_accuracy:
+          description: >-
+            Declaration of timestamp accuracy, which is the one-sided width of the 95%-confidence interval around `timestamp` for the true time of applicability for any of the data fields.
+          type: number
+          format: float
+          minimum: 0
+          example: 1
+          default: 0
+        position:
+          $ref: '#/components/schemas/AircraftPosition'
+        operational_status:
+          $ref: '#/components/schemas/OperationalStatus'
+        track:
+          description: Direction of flight expressed as a "True North-based" ground
+            track angle.  This value is provided in degrees East of North.  A value of
+            360 indicates that the true value is unavailable.
+          type: number
+          format: float
+          maximum: 360
+          minimum: 0
+          example: 295
+          default: 0
+        speed:
+          description: Ground speed of flight in meters per second.
+          type: number
+          format: float
+          minimum: 0
+          example: 1.9
+          default: 0
+        speed_accuracy:
+          description: Accuracy of horizontal ground speed.
+          anyOf:
+            - $ref: '#/components/schemas/SpeedAccuracy'
+        vertical_speed:
+          description: Geodetic vertical speed upward.  Units of meters per second.
+          type: number
+          format: float
+          example: -0.2
+          default: 0
+    OperationalStatus:
+      description: >-
+        Indicates operational status of associated aircraft.
+        
+        * `Undeclared`: The system does not support acquisition of knowledge about the status of the aircraft.
+        
+        * `Ground`: The aircraft is reporting status but is not airborne.
+        
+        * `Airborne`: The aircraft is, or should be considered as, being airborne.
+        
+        * `Emergency`: The aircraft is reporting an emergency.
+        
+        * `Unknown`: The system supports acquisition of knowledge about the status of the aircraft, but the status cannot currently be determined.
+      enum:
+        - Undeclared
+        - Ground
+        - Airborne
+        - Emergency
+        - Unknown
+      type: string
+      example: Airborne
+      default: Undeclared
+    AircraftPosition:
```

Comment:
Should we re-use/reference the utm.yaml for VehicleTelemetry? https://github.com/astm-utm/Protocol/blob/cb7cf962d3a0c01b5ab12502f5f54789624977bf/utm.yaml#L1232

Reply:
No, certainly not -- this interface is providing inputs to the USS-under-test's user interface.  There is no reason those inputs would exactly match the format of ASTM F3548-21, and we need to be able to iterate on this interface independent of standards development.  We actually already made the mistake of tying the original RID injection interface to F3411-19 data objects and we were forced to fix that mistake when we needed to test some things that were different in F3411-22a.

Additionally, some standard-defined data elements may not be feasible to obtain from the user (instead must be calculated or inferred from user-provided data) and in those cases we would not want to inject something that could not actually be provided to the true system via the normal user interface.